pubtools/pulplib/_impl/client/client.py

```diff
@@ -109,6 +109,9 @@ def __init__(self, url, **kwargs):
 
                 These may be used, for example, to configure the credentials
                 for the Pulp server or to use an alternative CA bundle.
+
+            task_throttle
+                If passed in kwargs, it'll be used as param to _task_executor.
```

Comment:
I don't think the docs make sense as the reader of the docs is not supposed to know what a private variable like _task_executor is.

Here's a suggestion:

```suggestion
            int task_throttle
                Maximum number of queued or running tasks permitted for this client.
                If more than this number of tasks are running, the client will wait before triggering more.
                This can be used to ensure no single client overwhelms the Pulp server.
```

I'd like to suggest also moving the doc for this one above the requests.Session parameters because it's more pulp-specific while the requests.Session stuff is generic.  Docs seem to flow better than way.